matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/auth/converter.kt

```diff
@@ -1,127 +1,129 @@
-                        }
-                        defaultLanguage -> {
-                            // We found default language
-                            defaultLanguageUrlAndName = extractUrlAndName(policy[policyKey])
-                        }
-                        else            -> {
-                            if (firstUrlAndName == null) {
-                                // Get at least some data
-                                firstUrlAndName = extractUrlAndName(policy[policyKey])
-                            }
-                        }
-                    }
-                }
-
-                // Copy found language data by priority
-                when {
-                    userLanguageUrlAndName != null    -> {
-                        localizedFlowDataLoginTermsLocalizedUrl = userLanguageUrlAndName!!.url
-                        localizedFlowDataLoginTermsLocalizedName = userLanguageUrlAndName!!.name
-                    }
-                    defaultLanguageUrlAndName != null -> {
-                        localizedFlowDataLoginTermsLocalizedUrl = defaultLanguageUrlAndName!!.url
-                        localizedFlowDataLoginTermsLocalizedName = defaultLanguageUrlAndName!!.name
-                    }
-                    firstUrlAndName != null           -> {
-                        localizedFlowDataLoginTermsLocalizedUrl = firstUrlAndName!!.url
-                        localizedFlowDataLoginTermsLocalizedName = firstUrlAndName!!.name
-                    }
-                }
-            }
-
-            result.add(LocalizedFlowDataLoginTerms(
-                    policyName = localizedFlowDataLoginTermsPolicyName,
-                    version = localizedFlowDataLoginTermsVersion,
-                    localizedUrl = localizedFlowDataLoginTermsLocalizedUrl,
-                    localizedName = localizedFlowDataLoginTermsLocalizedName
-            ))
-        }
-    }
-
-    return result
-}
-
-private fun extractUrlAndName(policyData: Any?): UrlAndName? {
-    if (policyData is Map<*, *>) {
-        val url = policyData["url"] as String?
-        val name = policyData["name"] as String?
-
-        if (url != null && name != null) {
-            return UrlAndName(url, name)
-        }
-    }
-    return null
-}
+/*
+ * Copyright 2020 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.android.sdk.api.auth
```

Comment:
wasn't expecting such a large diff for this file, I'm guessing the line separator character has been switched to the project define one :thinking:

Reply:
It must be so!